android/app/google-services.json

```diff
@@ -14,6 +14,14 @@
         }
       },
       "oauth_client": [
```

Comment:
Is this related to the G.Drive permission?

Reply:
Google libraries are needed for the notification library, here only sha1 is added to the application